debezium-storage/debezium-storage-redis/src/main/java/io/debezium/storage/redis/offset/RedisOffsetBackingStore.java

```diff
@@ -92,6 +118,7 @@ private void load() {
                         })
                 .await().indefinitely();
         this.data = new HashMap<>();
+        LOGGER.info("Offsets are found: {}", !offsets.isEmpty());
```

Comment:
I'd recommend to print offset content instead of just true/false flag. It might be useful in future diagnostics.

Reply:
Done